README.MD

```diff
@@ -28,10 +27,19 @@ import { render } from 'react-dom'
 import { createStore, combineReducers } from 'redux';
 
 // 1. Connect your data, similar to react-redux @connect
+// Decorator version
 @asyncConnect({
-  lunch: (params, helpers) => Promise.resolve({id: 1, name: 'Borsch'})
+  lunch: (params, store, helpers) => Promise.resolve({id: 1, name: 'Borsch'})
```

Comment:
ah i thought it was a mistake. How come the api is different for the decorator and the static method?
Actually i'm not sure i see the point of return a value to the decorator, if you're going to to use redux, the data will be fetched by `connect` automatically when the reducers get updated